x/stakeibc/keeper/icacallbacks_detokenize_test.go

```diff
@@ -14,12 +15,12 @@ import (
 // to the callback
 func (s *KeeperTestSuite) SetupTestDetokenizeCallback() []byte {
 	// Store the LSMDeposit record with status DETOKENIZATION_IN_PROGRESS
-	deposit := types.LSMTokenDeposit{
+	deposit := recordstypes.LSMTokenDeposit{
 		ChainId: HostChainId,
 		Denom:   LSMTokenBaseDenom,
-		Status:  types.DETOKENIZATION_IN_PROGRESS,
+		Status:  recordstypes.LSMTokenDeposit_DETOKENIZATION_IN_PROGRESS,
 	}
-	s.App.StakeibcKeeper.SetLSMTokenDeposit(s.Ctx, deposit)
+	s.App.RecordsKeeper.SetLSMTokenDeposit(s.Ctx, deposit)
```

Comment:
I assume this works when you ran the tests but just wondering why here in the stakeibc module why don't we have to refer to this as s.App.StakeibcKeeper.RecordsKeeper since everywhere else in this module we have to go one level in to get to the records keeper?

Reply:
The `App` from the unit tests has access to everything and all keepers.
Whereas, when we're in the stakeibc module, we only have access to the keepers that are dependencies of stakeibc [[here](https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/keeper/keeper.go#L34-L50)]

Reply:
Good to know for testing